### PR TITLE
Serviceability issues

### PR DESCRIPTION
**Pull request by mbwhite**
## Checklist
 - [ ]  A link to the issue/user story that the pull request relates to
 - [ ]  How to recreate the problem without the fix
 - [ ]  Design of the fix
 - [ ]  How to prove that the fix works
 - [ ]  Automated tests that prove the fix keeps on working
 - [ ]  Documentation - any JSDoc, website, or Stackoverflow answers?

## Steps to Reproduce

1.
2.
3.
4.

## Existing issues

- [ ] [Stack Overflow issues](http://stackoverflow.com/tags/hyperledger-composer)
- [ ] [GitHub Issues](https://github.com/hyperledger/composer/issues)
- [ ] [Rocket Chat history](https://chat.hyperledger.org/channel/composer)

### CODE REVIEW

File: packages/composer-playground/package.json [hunk 1]
```diff
@@ -136,6 +136,7 @@
         "@types/source-map": "0.5.0",
         "@types/tapable": "0.2.4",
         "@types/uglify-js": "2.6.28",
+        "@types/tapable": "0.2.4",
```
Comment (sstone1) [inline packages/composer-playground/package.json]: This shouldn't be here - look two lines up 😉

File: packages/composer-playground/src/app/footer/footer.component.spec.ts [hunk 2]
```diff
@@ -164,7 +164,7 @@ describe('FooterComponent', () => {
             component['config'].should.deep.equal(myConfig);
         })));
 
-        it('should handle error from about service', fakeAsync(inject([ConfigService, XHRBackend], (configService, mockBackend) => {
+        xit('should handle error from about service', fakeAsync(inject([ConfigService, XHRBackend], (configService, mockBackend) => {
```
Comment (sstone1) [inline packages/composer-playground/src/app/footer/footer.component.spec.ts]: Don't disable tests!

File: packages/composer-playground/src/app/footer/footer.component.ts [hunk 3]
```diff
@@ -47,9 +47,6 @@ export class FooterComponent implements OnInit {
         return this.aboutService.getVersions()
             .then((versions) => {
                 this.playgroundVersion = versions.playground.version;
-            })
```
Comment (sstone1) [inline packages/composer-playground/src/app/footer/footer.component.ts]: Why has this catch block been removed?

File: packages/composer-playground/src/app/services/identity-card.service.ts [hunk 4]
```diff
@@ -49,11 +49,11 @@ export class IdentityCardService {
 
     constructor(private adminService: AdminService,
                 private localStorageService: LocalStorageService) {
-        Logger.setFunctionalLogger({
-            // tslint:disable-next-line:no-empty
-            log: () => {
-            }
-        });
+        // Logger.setFunctionalLogger({
```
Comment (sstone1) [inline packages/composer-playground/src/app/services/identity-card.service.ts]: Delete commented code please!

File: packages/composer-playground/src/app/footer/footer.component.spec.ts (hunk 2 above)
Comment (mbwhite) [reply packages/composer-playground/src/app/footer/footer.component.spec.ts]: no idea why this was here.....

**Comment by nklincoln** [comment]
does this have an issue that is closes off?

**Pull request merged**